GVFS/GVFS.Common/Maintenance/GitMaintenanceStep.cs

```diff
@@ -196,8 +199,10 @@ protected void LogErrorAndRewriteMultiPackIndex(ITracer activity, GitProcess.Res
             string multiPackIndexPath = Path.Combine(this.Context.Enlistment.GitPackRoot, "multi-pack-index");
             errorMetadata["TryDeleteFileResult"] = this.Context.FileSystem.TryDeleteFile(multiPackIndexPath);
 
-            GitProcess.Result rewriteResult = this.RunGitCommand((process) => process.WriteMultiPackIndex(this.Context.Enlistment.GitObjectsRoot));
+            GitProcess.Result rewriteResult = this.RunGitCommand((process) => process.WriteMultiPackIndex(this.Context.Enlistment.GitObjectsRoot), "WriteMultiPackIndex");

```

Comment:
Minor: Here and below, switch from string constants (e.g. "WriteMultiPackIndex") to `nameof` (e.g. `nameof(process.WriteMultiPackIndex)`.

This comment applies in a few different spots/files.